docs/en_US/Compressor/AutoCompression.md

```diff
@@ -9,13 +9,13 @@ You can easily compress a model with NNI compression. Take pruning for example,
 ```python
 from nni.compression.torch import LevelPruner
 config_list = [{ 'sparsity': 0.8, 'op_types': ['default'] }]
-pruner = LevelPruner(config_list)
-pruner(model)
+pruner = LevelPruner(model, config_list)
```

Comment:
Logically a pruner should be a standalone component. It should not need a model to "create" a pruner. So this looks strange.
But I known we don't have a better solution yet.

Reply:
yes, let's keep it for now